requirements/test-requirements.in

```
@@ -13,3 +13,4 @@ sqlalchemy-postgres-copy==0.5.0
 flaky==3.7.0
 freezegun~=1.1
 radon
+flake8-polyfill  # Needed only for a bug in radon's dependency management
```

```suggestion
radon[flake8]
```
I didn't know this was the convention, but changing this to `radon[flake8]` will result in radon getting installed *with the flake8 extra*.  This will resolve the dependency issue because pip will install `flake8-polyfill` as per radon's setup.py.

I still think radon has something wrong, because it's rude IMO for a library to break other projects who don't explicitly ask for their "extras".  I need to do some more research on this. In the meantime, I would recommend this format.

---

I found [this SA answer](https://stackoverflow.com/questions/41268863/difference-between-extras-require-and-install-requires-in-setup-py/45043494#45043494) quite helpful.

---

Removed my request for changes here. Sorry, that was a bit severe for a perfectly suitable solution. I'm fine with either way.

---

Works for me.  Just merged that suggestion and rebuilt requirements.  Glad you pinged me actually - I thought I'd merged this and it dropped off my radar